README.md

```diff
@@ -56,33 +56,52 @@ xcode-select --install
 
 ### Installing FlowForge
 
-1. Create a directory to be the home of your FlowForge install. For example: `/opt/flowforge`
+1. Create a directory to be the home of your FlowForge install. For example: `/opt/flowforge` or `c:\flowforge`
 
    For Linux/MacOS:
-
     ```
     sudo mkdir /opt/flowforge
     sudo chown $USER /opt/flowforge
     ```
 
+   For Windows:
+    ```console
+    mkdir c:\flowforge
+    ```
+
 2. Download the Installer zip file from https://github.com/flowforge/installer/releases
 
 3. Unzip the downloaded file into a temporary location and copy its contents to
    the FlowForge directory
 
+   For Linux/MacOS:
    ```
     cd /tmp/
     unzip flowforge-installer-x.y.z.zip
     cp -R flowforge-install-x.y.z/* /opt/flowforge
+   ```
+
+   For Windows:
+   ```console
+    cd c:\temp
+    tar -xf flowforge-installer-x.y.z.zip
```

Comment:
Shouldn't this be unzip not tar?

Reply:
no, windows does not have (by default) unzip but `tar` is built in Win 10 build 17063 or later. 
the command works as written (tested)

Reply:
OK, (it does have built in unzip in file explorer)
In that case for the 0.2 release we might move over to using a tar file as there are option to strip the leading dir off the output and we can do away with the temp directory.

Reply:
> OK, (it does have built in unzip in file explorer)

unfortunately that is only accessible via powershell or vbs

> In that case for the 0.2 release we might move over to using a tar file as there are option to strip the leading dir off the output and we can do away with the temp directory.

Sounds good.

Q. how would you normally capture this sort of thing so it doesnt get missed?

Reply:
It should be tracked as an issue #25